crates/fluence-app-service/src/config.rs

```diff
@@ -14,16 +14,17 @@
  * limitations under the License.
  */
 
-use marine::MarineConfig;
+use marine::generic::MarineConfig;
```

Comment:
As `AppService` is also generic now, there is a question: keep wasmtime and generic versions of exported types in marine-runtime and marine-core or leave only generics ones?

Reply:
Decided to make it in `fluence-app-service` just as in  `marine-runtime` and `marine-core`: make export wasmtime version by default, and generic types in `generic` module